Tests/UITests/TestApp/Resources/Localizable.xcstrings

```diff
@@ -1,6 +1,9 @@
 {
   "sourceLanguage" : "en",
   "strings" : {
+    "Key" : {
+      "extractionState" : "manual"
```

Comment:
When building, Xcode automatically extracts all required `String`s from the Swift files, so no need to `manual`ly extract them.